js/playlist.js

```diff
@@ -187,6 +187,9 @@ function(ytData,ytIframe,template,$) {
         savePlaylist();
     }
 
+
+
+
```

Comment:
Unnecessary change? please remove this. It is always good to check your changes using `git diff` before pushing!

Reply:
Sorry for that.